Sources/ITwinMobile/ITMMessenger.swift

```diff
@@ -21,6 +21,19 @@ internal extension JSONSerialization {
         return string(withITMJSONObject: object, prettyPrint: false)
     }
 
+    /// Determines if value is a serializable type, i.e., one that can be passed into JSONSerialization.data(withJSONObject:)
+    /// - Parameter value: value to check
+    /// - Returns: true if value is a Dictionary or Array, false otherwise.
+    static func isSerializableType(_ value: Any) -> Bool {
```

Comment:
From the description of this function is seems like it is trying to accomplish the same thing as JSONSerialization.isValidJSONObject. If it isn't the same then I think some clarification of the documentation is needed here.

Reply:
I was unaware of that function. I switch to it and test to verify it works.

Reply:
Fixed.